modin/pandas/base.py

```diff
@@ -3602,30 +3415,6 @@ def __array__(self, dtype=None):
         arr = self.to_numpy(dtype)
         return arr
 
-    def __array_wrap__(self, result, context=None):
```

Comment:
why removed?

Reply:
This operation has been removed in https://pandas.pydata.org/docs/whatsnew/v2.0.0.html.